webpack.config.js

```diff
@@ -12,6 +16,66 @@ module.exports = (webpackConfigEnv, argv) => {
   });
 
   return merge(defaultConfig, {
+    entry: {
+      common: path.join(__dirname, 'lib', 'mtfh-common.tsx'),
+      'common/lib/auth': path.join(__dirname, 'lib', 'auth'),
+      'common/lib/http': path.join(__dirname, 'lib', 'http'),
+      'common/lib/config': path.join(__dirname, 'lib', 'config'),
+      'common/lib/configuration': path.join(__dirname, 'lib', 'configuration'),
+      'common/lib/components': path.join(__dirname, 'lib', 'components'),
+      'common/lib/hooks': path.join(__dirname, 'lib', 'hooks'),
+      'common/lib/utils': path.join(__dirname, 'lib', 'utils'),
+      'common/lib/api/person/v1': path.join(
```

Comment:
```tsx
'common/lib/api/person/v1': path.join(apiPath, 'person', 'v1')
```
Might help get everything on the same line

Reply:
ah nice!